### DISCUSSION TITLE
ArrangoDb new license impact ?

### DISCUSSION THREAD

**Discussion by guitarzero** [yeti-platform/yeti#1176] [General]
Hi,
Thank you for this great project. 

If I understood correctly (not sure BTW): 
- At Hack.lu 2023, @tomchop mentionned that the Yeti project had been motivated by AGPL-v3 licenses issues (among other concerns). 
- Upcoming versions of Arrango Db (>3.11) are governed by another license making upgrades impossible for commercial purposes or 100GB+ storage on a single cluster. 

Does this change in licensing model have any consequence for Yeti future development ? 

Thank you
Stéphane

**Comment by sebdraven** [comment]
Hi,

for the moment we have fixed ArangoDB version 3.11. And we keep on the development  on the same model.

Two solutions are possible:

- a fork of ArangoDB is created since 3.11 and we change the backend to this fork
- We changes the backend totally with a new database

**Reply by guitarzero** [reply]
Thank you very much for this quick answer !!
Have a nice day
